azure-pipelines.yml

```diff
@@ -208,26 +208,6 @@ jobs:
       includePatterns:
         - "/sw/device/***"
 
-- job: chip_earlgrey_verilator
-  displayName: Build Verilator simulation of the Earl Grey toplevel design
-  dependsOn: lint
-  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
-  pool: ci-public
-  steps:
-  - template: ci/install-package-dependencies.yml
-  - bash: |
-      python3 --version
-      fusesoc --version
-      verilator --version
-      verible-verilog-lint --version
-    displayName: Display environment
-  - bash: ci/scripts/build-chip-verilator.sh earlgrey
```

Comment:
If it's no longer used, could you delete this script?

Reply:
I'm going to work on migrating earlgrey in a separate PR, so I'll leave this in for now.